decidim-comments/app/frontend/comments/add_comment_form.component.tsx

```diff
@@ -11,7 +11,7 @@ const { I18n, Translate } = require("react-i18nify");
 import {
   AddCommentFormCommentableFragment,
   AddCommentFormSessionFragment,
-  AddCommentMutation,
+  addCommentMutation,
```

Comment:
The `schema.ts` is generated using `apollo-codegen`. It looks like they changed this to mirror graphql conventions but I am not sure 😄